tensorboard/plugins/pr_curve/tf_pr_curve_dashboard/tf-pr-curve-dashboard.html

```diff
@@ -299,6 +278,35 @@ <h3>No precision–recall curve data was found.</h3>
         const defaultDisplayName = tag.replace(/\/pr_curves$/, '');
         return tf_utils.aggregateTagInfo(runToTagInfo, defaultDisplayName);
       },
+      _createDataChangeCallback(tag) {
+        return function onDataChange(runToData) {
```

Comment:
Do you preferred named function for better debugging? Arrow function may work better since you use `this`.

Reply:
You’re right; that should definitely work. (I was being a bit cautious
with the receiver because I don’t know exactly how Polymer binds `this`
in all the cases that matter, but you’re of course right that if an
explicit `.bind(this)` works then an arrow function should, too, so that
was silly of me.) Done.